torch/jit/mobile/__init__.py

```diff
@@ -0,0 +1,86 @@
+import torch._C
+import torch._jit_internal as _jit_internal
+import torch.jit.annotations
+
+from torch.nn import Module
+
+import pathlib
+from torch._six import string_classes
+from torch.serialization import validate_cuda_device
+
+import os
+
+def load_for_lite_interpreter(f, map_location=None):
```

Comment:
Can you put an underscore prefix on this so it doesn't become an externally-supported API?

Reply:
sure.